app/spike/internal/cmd/secret/get.go

```diff
@@ -73,14 +77,66 @@ func newSecretGetCommand(source *workloadapi.X509Source) *cobra.Command {
 				return
 			}
 
-			data := secret.Data
-			for k, v := range data {
-				fmt.Printf("%s: %s\n", k, v)
-			}
+			printSecretResponse(secret)
 		},
 	}
 
 	getCmd.Flags().IntP("version", "v", 0, "Specific version to retrieve")
+	getCmd.Flags().BoolP("metadata", "m", false, "Show metadata instead of secret values")
```

Comment:
Instead of a `-m` flag, can we make it a "metadata" subcommand.

As in:

```bash
spike metadata get <path>
```

This way, it will look similar to Vault's kv metadata API, and it will be less confusing for the end-user.

ref:
https://developer.hashicorp.com/vault/docs/commands/kv/metadata